torchtitan/utils.py

```diff
@@ -17,6 +17,11 @@
 from torchtitan.parallelisms import ParallelDims
 
 
+def move_to_empty(model_parts: List[torch.nn.Module], device: torch.device):
+    for model in model_parts:
+        model.to_empty(device="cuda")
```

Comment:
We probably should simply do this in `train.py` as well. It's similar to other for-loops in `train.py`, which are only called once in the code base.